src/bp/core/services/dialog/dialog-engine.ts

```diff
@@ -223,6 +228,57 @@ export class DialogEngine {
     }
   }
 
+  private copyVarsToParent(childFlow: FlowView, callerNode: FlowNode, event: IO.IncomingEvent) {
+    const { workflows } = event.state.session
+    const { createVariable } = event.state
```

Comment:
not fan of `createVariable` being in the state
I know this is from @allardy but can you add a TODO to move this to a helper before official release?